test/integration/scenarios/03.basicWithNsDelete/manifest/oss/solution.yaml

```diff
@@ -12,7 +12,7 @@ spec:
   - name: prometheus-server
     type: container
     properties:
-      container.ports: "[{\"containerPort\":9090,\"protocol\":\"TCP\"}]"
+      container.ports: "[{\"containerPort\":${{$secret('container','port')}},\"protocol\":\"TCP\"}]"
```

Comment:
Do we need precheck for the secret reference in webhook? Or wait till instance reconciliation as catalog config?

Reply:
it should be just like catalog config